### ISSUE TITLE
Using target="_blank" per documentation results in a build error

### ISSUE THREAD

**Issue by CitizenBeta** [ionic-team/ionic#18690]
**Ionic version:**
[x] **4.x**

**I'm submitting a ...** 
[x] bug report
[ ] feature request

**Current behavior:**
Using [target]="'_blank'" per the documentation results in a build error.

**Expected behavior:**
Per the documentation it should open the 

**Steps to reproduce:**
Add [target]="'_blank'" to an ion-button component.

**Related code:**

<ion-button href="https://github.com" target="_blank">

**Ionic info:** 

```
Ionic:

   Ionic CLI                     : 5.2.1 (/usr/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.4.2
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v10.16.0 (/usr/bin/node)
   npm    : 6.9.0
   OS     : Linux 5.0```

**Comment by liamdebeasi** [comment]
Thanks for the issue! What is the build error you are receiving?

**Comment by CitizenBeta** [comment]
Sorry, console error, not build error:

> ERROR Error: Uncaught (in promise): Error: Template parse errors:
> Can't bind to 'target' since it isn't a known property of 'ion-button'.
> 1. If 'ion-button' is an Angular component and it has 'target' input, then verify that it is part of this module.

href works, rel works, target does not.

**Comment by brandyscarney** [comment]
Please update to the latest Ionic. The `target` property was not included until `4.6.0`: https://ionicframework.com/docs/release-notes
